### ISSUE TITLE
Keyboard navigation lock in Pie Chart

### ISSUE THREAD

**Issue by marilia-gb** [recharts/recharts#6921]
## What is the current behavior?

I used 3 charts: pie, line, and bar. The charts receive keyboard focus (by pressing the tab key), and it's also possible to switch between internal items (by pressing the right and left arrow keys).

However, navigating the internal items of the pie chart only works after hovering the mouse over the chart at least once. Without using the mouse, navigation with the arrow keys does not work.

[Video](https://drive.google.com/file/d/18Y3IYU6XUioN6JzB2iv9_mlu-mApvQbB/view?usp=sharing)

## What is the expected behavior?

I hope that in the pie chart's onFocus event, it will be possible to navigate using the arrows and without the need to configure a defaultIndex value for the Tooltip.

## Please provide a demo of the problem in a sandbox

https://recharts.github.io/storybook-static/iframe.html?globals=&args=&id=api-chart-piechart--api&viewMode=story

Remove `defaultIndex` from Tooltip, and try navigation use only keyboard.

## Which versions of Recharts, and which browser / OS are affected by this issue? Did this work in previous versions of Recharts?

"recharts": "3.6.0"

**Comment by Jeevanm2004** [comment]
Hi, I'd like to work on this issue. I'll investigate the PieChart keyboard navigation and submit a fix.

**Comment by marilia-gb** [comment]
Thank so much! It's working 🚀
